kof-operator/internal/controller/regional_cluster_configmap_controller.go

```diff
@@ -0,0 +1,60 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=core,resources=configmaps/status,verbs=get;update;patch
+type RegionalClusterConfigMapReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+// Make controller react to `ConfigMaps` having the expected label only.
+func (r *RegionalClusterConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewControllerManagedBy(mgr).
+		Named("regional-cluster-config-map").
+		For(&corev1.ConfigMap{}).
+		WithEventFilter(predicate.NewPredicateFuncs(func(obj client.Object) bool {
+			labels := obj.GetLabels()
+			role, ok := labels[KofClusterRoleLabel]
+			return ok && role == KofRoleRegional
```

Comment:
Let's keep it simple because `KofRoleRegional` is not an empty string:
```suggestion
			return obj.GetLabels()[KofClusterRoleLabel] == KofRoleRegional
```